modules/jdbc/src/main/java/org/apache/ignite/jdbc/IgniteJdbcDriver.java

```diff
@@ -29,11 +29,104 @@
 import java.util.Properties;
 import java.util.logging.Logger;
 import org.apache.ignite.internal.client.proto.ProtocolVersion;
+import org.apache.ignite.internal.jdbc.ConnectionPropertiesImpl;
+import org.apache.ignite.internal.jdbc.JdbcConnection;
 
 /**
- * JDBC driver thin implementation for Apache Ignite 3.x.
+ * JDBC driver implementation for Apache Ignite 3.x.
  *
- * <p>Driver allows to get distributed data from Ignite Data Storage using standard SQL queries and standard JDBC API.
+ * <p>Driver allows to get distributed data from Ignite 3 Data Storage using standard SQL queries and standard JDBC API.</p>
+ * <h2>Register the JDBC drivers</h2>
+ * <p>
+ * The JDBC driver registration is automatically done via the Java Standard Edition Service Provider mechanism.
+ * Ignite JDBC driver implements this feature and it is automatically registered for case the JDBC driver jar presents in classpath.
+ * </p>
```

Comment:
wrong formatting of paragraphs. Please refer to [guidelines](https://google.github.io/styleguide/javaguide.html#s7.1.2-javadoc-paragraphs) for details. Besides, common practice is to omit closing tag for paragraph. `<h*>` tags are block tags as well, thus must follow the same rule

Reply:
fixed